packages/editor/src/client/atoms/textarea.tsx

```diff
@@ -0,0 +1,10 @@
+import React, {forwardRef} from 'react'
```

Comment:
Textarea are now done via accepter: https://rsuitejs.com/components/form/
Copied the component from the code examples and added the props.

There's probably a better way instead of having to manually declare the TextareaProps. I tried  `React.TextareaHTMLAttributes<unknown>` and this didn't work as the rsuite defines the interface a bit different.